index.html

```diff
@@ -310,7 +310,7 @@ <h3 class="table-header">Friday, October 29th</h3>
             <tr>
                 <td class="left">Team Formation</td>
                 <td class="middle">Dough 416</td>
-                <td class="right">7 pm</td>
+                <td class="right">7 Pm</td>
```

Comment:
Should we keep fully capitalized (like PM) or fully lowercase (pm) @hunterprice04 @Manan-dev